pwa/src/templates/componentDetail/ComponentsDetailTemplate.tsx

```diff
@@ -25,22 +25,32 @@ import { GitHubLogo } from "../../assets/svgs/GitHub";
 import { RatingIndicatorTemplate } from "../templateParts/ratingIndicator/RatingIndicatorTemplate";
 import { Tag } from "../../components/tag/Tag";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
-import { faLayerGroup, faScroll } from "@fortawesome/free-solid-svg-icons";
+import { faHouse, faInfoCircle, faLayerGroup, faRepeat, faScroll } from "@fortawesome/free-solid-svg-icons";
 import _ from "lodash";
+import { ToolTip } from "../../components/toolTip/ToolTip";
+import { categories, TCategories } from "../../data/categories";
+import { categories as _categories } from "../../data/filters";
 
 interface ComponentsDetailTemplateProps {
   componentId: string;
 }
 
 export const ComponentsDetailTemplate: React.FC<ComponentsDetailTemplateProps> = ({ componentId }) => {
-  const [currentTab, setCurrentTab] = React.useState<number>(0);
   const { t } = useTranslation();
+  const [currentTab, setCurrentTab] = React.useState<number>(0);
 
   const queryClient = new QueryClient();
 
   const _useComponent = useComponent(queryClient);
   const _getComponent = _useComponent.getOne(componentId);
 
+  const layer: TCategories = t(_.upperFirst(_getComponent.data?.embedded?.nl.embedded.commonground.layerType));
```

Comment:
If we'd instead use `_.toLower()` here, could we get away with _not_ changing the `TCategories` keys to uppercase? Would be more consistent with our naming conventions.